README.md

```diff
@@ -921,6 +925,14 @@ params.PackageType = "maven"
 err := servicesManager.GetAllRepositoriesFiltered(params)
 ```
 
+#### Check if Repository Exists
+
+You can check if a repository is exist in Artifactory:
```

Comment:
You can check if a repository is exist in Artifactory:
-->
You can check whether a repository exists in Artifactory: